src/custom/hooks/useRecentActivity.ts

```diff
@@ -0,0 +1,93 @@
+import { useMemo } from 'react'
+import { isTransactionRecent, useAllTransactions } from 'state/transactions/hooks'
+import { useOrders } from 'state/orders/hooks'
+import { useActiveWeb3React } from 'hooks'
+import { Order, OrderStatus } from 'state/orders/actions'
+import { TransactionDetails } from '@src/state/transactions/reducer'
+
+type TransactionAndOrder =
+  | (Order & { addedTime: number })
+  | (TransactionDetails & {
+      id: string
+      status: OrderStatus
+    })
+
+// One fill day in MS
+const DAY_MS = 86_400_000
```

Comment:
👌

Reply:
this is how uni deals with recents, they use last day transactions and so am doing so for orders as well
basically tried to match their logic as close as possible